### PR TITLE
Move synchronization CLI to cloud nataive

### PR DESCRIPTION
**Pull request by steveoh**
This will now be a scheduled cloud run.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      - name: 🏷️ Extract tags from GitHub
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: gcr.io/${{ secrets.PROJECT_ID }}/app
+          tags: |
+            type=ref,suffix=-{{sha}},event=branch
+            type=ref,prefix=pr-,suffix=-{{sha}},event=pr
+            type=semver,pattern={{version}}
+            latest
```
Comment (stdavis) [inline .github/workflows/deploy.yml:55]: This looks nicer than our previous hacks!

**Review by stdavis — COMMENTED** [review]
Just had one question...

File: src/cloudb/roles.py [hunk 2]
```diff
@@ -32,7 +32,7 @@ def create_read_only_user(schemas):
                         NOCREATEDB
                         NOCREATEROLE
                         NOINHERIT
-                        NOLOGIN
+                        NOloggingIN
```
[the remark this answers is not in the document]
Comment (steveoh) [reply src/cloudb/roles.py:35]: nope, that's a multiple cursor error...

Comment (steveoh) [inline src/cloudb/roles.py:35]:
```suggestion
                        NOLOGIN
```

**Pull request merged**